test/lib/builder/builder.js

```diff
@@ -53,14 +53,16 @@ function cloneProjectTree(tree) {
 async function checkFileContentsIgnoreLineFeeds(expectedFiles, expectedPath, destPath) {
 	for (let i = 0; i < expectedFiles.length; i++) {
 		const expectedFile = expectedFiles[i];
-		const relativeFile = path.relative(expectedPath, expectedFile);
-		const destFile = path.join(destPath, relativeFile);
-		const currentFileContentPromise = readFile(destFile, "utf8");
-		const expectedFileContentPromise = readFile(expectedFile, "utf8");
-		const assertContents = ([currentContent, expectedContent]) => {
-			assert.equal(currentContent.replace(newLineRegexp, "\n"), expectedContent.replace(newLineRegexp, "\n"));
-		};
-		await Promise.all([currentFileContentPromise, expectedFileContentPromise]).then(assertContents);
+		if (!expectedFile.endsWith(".zip")) {
```

Comment:
Is this change still needed? The builder tests don't produce zip files anymore, now that the task is not executed by default anymore.

Reply:
Good catch. Removed!